guides/release/components/arguments-and-attributes.md

```diff
@@ -263,6 +263,17 @@ Results in:
 </p>
 ```
 
+#### Note About Some `aria` Attributes
```

Comment:
What about just the following?

```suggestion
#### `aria` Attributes
```

Reply:
I'll try to re-work the content to include all aria attributes.